addons/escoria-core/game/core-scripts/esc/esc_action_manager.gd

```diff
@@ -570,6 +572,19 @@ func perform_inputevent_on_object(
 				)
 
 
+# Determines whether the object in question can be acted upon.
+#
+# #### Parameters
+#
+# - global_id: the global ID of the item to examine
+#
+# *Returns* True iff the item represented by global_id can be acted upon.
```

Comment:
```suggestion
# *Returns* True if the item represented by global_id can be acted upon.
```

Reply:
Won't fix. "iff" is short for "if and only if" and has a very precise meaning in the language of logic.